cmd/convox/env_test.go

```diff
@@ -160,7 +160,7 @@ func TestEnvSetStdin(t *testing.T) {
 		test.Http{
 			Method: "POST",
 			Path:   "/apps/myapp/environment",
-			Body:   "foo=bar\nping=pong\n",
+			Body:   "# Comment followed by blank line\n\nfoo=bar\nping=pong\n",
```

Comment:
This line should be reverted back as `foo=bar\nping=pong\n` is what's returned by the test server.